packages/run-protocol/src/vaultFactory/liquidateMinimum.js

```diff
@@ -24,24 +24,26 @@ const trace = makeTracer('LM');
 const start = async zcf => {
   const { amm } = zcf.getTerms();
 
-  /**
-   * @param {Amount<'nat'>} runDebt
-   */
-  const makeDebtorHook = runDebt => {
-    const runBrand = runDebt.brand;
-    return async debtorSeat => {
+  const makeDebtorHook = _runDebt => {
+    /**
+     * @param {ZCFSeat} debtorSeat
+     * @param {object} root0
+     * @param {Amount<'nat'>} root0.debt
+     */
+    return async (debtorSeat, { debt: runDebt }) => {
+      const runBrand = runDebt.brand;
```

Comment:
why rename `debt`? especially when 'run' is going away
```suggestion
    return async (debtorSeat, { debt }) => {
      const debtBrand = debt.brand;
```

Reply:
I wanted to separate out the renaming through the rest of the code of `runDebt` to `debt` but it's minor.